var/spack/repos/builtin/packages/openssl/package.py

```diff
@@ -53,7 +53,8 @@ class Openssl(Package):
     version('1.0.1h', '8d6d684a9430d5cc98a62a5d8fbda8cf')
 
     depends_on("zlib")
-    # Also requires make and perl
+    depends_on("perl", type='build')
+    # Also requires make
```

Comment:
I think we can get rid of this "Also requires make" message. So do most of the packages in Spack.